network/network_linux.go

```diff
@@ -246,7 +246,7 @@ func isGreaterOrEqaulUbuntuVersion(versionToMatch int) bool {
 func readDnsInfo(ifName string) (DNSInfo, error) {
 	var dnsInfo DNSInfo
 
-	p := platform.NewExecClient()
+	p := platform.NewExecClient(logger)
```

Comment:
The same exec client we have for nm manager should be used here right ? similar to windows one?